swift/extractor/SwiftExtractor.cpp

```diff
@@ -36,11 +41,7 @@ static void archiveFile(const SwiftExtractorConfiguration& config, swift::Source
 
   std::error_code ec;
   fs::copy(source, destination, fs::copy_options::overwrite_existing, ec);
-
-  if (ec) {
-    std::cerr << "Cannot archive source file " << source << " -> " << destination << ": "
-              << ec.message() << "\n";
-  }
+  CODEQL_ASSERT(!ec, "Cannot archive source file {} -> {} ({})", source, destination, ec);
```

Comment:
This should not be an assertion, in case two processes try to archive the file, one of them fails, which is fine.